diff --git a/types/jsx.d.ts b/types/jsx.d.ts
index c742e9ee86f5a73b0704ed9a3fa0545e65cfdeb1..5653ae3da52922f34647257ff27d346f55e959d8 100644
--- a/types/jsx.d.ts
+++ b/types/jsx.d.ts
@@ -900,7 +900,7 @@ export namespace JSX {
     is?: string | undefined;
     inert?: boolean | undefined;
     lang?: string | undefined;
-    spellcheck?: boolean | undefined;
+    spellcheck?: boolean | "false" | "true" | undefined;
     style?: CSSProperties | string | undefined;
     tabindex?: number | string | undefined;
     title?: string | undefined;
